install_config/storage_examples/containerized_heketi_with_dedicated_gluster.adoc

```diff
@@ -0,0 +1,379 @@
+----
+
+. Register the `deploy-heketi` template:
++
+[source, bash]
+----
+$ oc create -f <template-path>/deploy-heketi-template
+----
+
+. Deploy the bootstrap Heketi container:
++
+[source, bash]
+----
+$ oc process deploy-heketi -v \
+         HEKETI_KUBE_NAMESPACE=<project-name> \
+         HEKETI_KUBE_APIHOST=<master-url-and-port> \
+         HEKETI_KUBE_INSECURE=y \
+         HEKETI_KUBE_USER=<cluster-admin-username> \
+         HEKETI_KUBE_PASSWORD=<cluster-admin-password> | oc create -f -
+----
+
+. Wait until the `deploy-heketi` pod starts and all services are running. Then get Heketi service details:
++
+[source, bash]
+----
+$ oc get svc
+NAME            CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+deploy-heketi   172.30.96.173   <none>        8080/TCP   2m
+----
+
+. Check if Heketi services are running properly, it must return `Hello from Heketi`.
++
+[source, bash]
+----
+$ curl http://<cluster-ip>:8080/hello
+Hello from Heketi
+----
+
+. Set an environment variable for the Heketi server:
++
+[source, bash]
+----
+$ export HEKETI_CLI_SERVER=http://<cluster-ip>:8080
+----
+
+[[dedicated-glusterfs-dynamic-example-loading-topology]]
+== Loading Topology
+
+Topology is used to tell Heketi about the environment and what nodes and devices
+it will manage.
+
+[NOTE]
+====
+Heketi is currently limited to managing raw devices only. If a device is already
+a Gluster volume, it is skipped and ignored.
+====
+
+. Create and load the topology file. There is a sample file located in
+*_/usr/share/heketi/toplogy-sample.json_* by default, or *_/etc/heketi_*
+depending on how it was installed.
++
+[source,json]
+----
+{
+  "clusters": [
+    {
+      "nodes": [
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster23.rhs"
+              ],
+              "storage": [
+                "192.168.1.200"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        },
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster24.rhs"
+              ],
+              "storage": [
+                "192.168.1.201"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        },
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster25.rhs"
+              ],
+              "storage": [
+                "192.168.1.202"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sde",
+            "/dev/sdf"
+          ]
+        },
+      ]
+    }
+  ]
+}
+----
+
+. Using `heketi-cli`, run the following command to load the topology of your
+environment.
++
+[source, bash]
+----
+$ heketi-cli topology load --json=topology.json
```

Comment:
\# heketi-cli topology load --json=topology.json
Error: Unable to open config file

The topology.json does existing, may some steps missing.